splitio/engine/filters/bloom_filter.py

```diff
@@ -0,0 +1,53 @@
+from splitio.engine.filters.base_filter import BaseFilter
+from splitio import util
+from bloom_filter2 import BloomFilter
+
+class ImpressionsBloomFilter(BaseFilter):
```

Comment:
rename to `BloomFilter` please

Reply:
BlooomFilter is the 3rd party class name used, did not want to have collisions